examples/lambdarank/_rank.train.position

```diff
@@ -0,0 +1,3005 @@
+pos_1

```

Comment:
Is testing the only reason this 3000+-line file is being included in the repo?

I think so, since you haven't also added `lambdarank_position_bias_regularizer` to https://github.com/microsoft/LightGBM/blob/master/examples/lambdarank/train.conf (and therefore the position bias factors will be ignored).

Please remove this and generate it dynamically from the unit test code, or explain why it should be checked into source control in the repo.

Reply:
Removed via fb0d251.